src/clusterfuzz/_internal/tests/core/bot/tasks/utasks/corpus_pruning_task_test.py

```diff
@@ -439,30 +435,20 @@ def tearDown(self):
   def test_prune(self):
     """Test pruning."""
     self._setup_env(job_type='libfuzzer_asan_job')
-    self.mock._record_cross_pollination_stats.side_effect = (
-        self.get_mock_record_compare(
-            project_qualified_name='test_fuzzer',
-            sources='test2_fuzzer',
-            initial_corpus_size=5,
-            corpus_size=3,
-            initial_edge_coverage=0,
-            edge_coverage=0,
-            initial_feature_coverage=0,
-            feature_coverage=0))
-
     uworker_input = uworker_io.DeserializedUworkerMsg(
         job_type='libfuzzer_asan_job', fuzzer_name='libFuzzer_test_fuzzer')
+
     corpus_pruning_task.utask_main(uworker_input)
 
     corpus_dir = os.path.join(self.temp_dir, 'corpus')
     os.mkdir(corpus_dir)
-    self.corpus.rsync_to_disk(corpus_dir)
 
+    self.corpus.rsync_to_disk(corpus_dir)
```

Comment:
Another thing that made this test nightmarish to debug was it's use of rsync_to_disk and rsync_from_disk, both in setup and here, which changed the results in subtle ways because of how I modified the functions.